src/main/groovy/com/athaydes/spockframework/report/vivid/VividAstInspector.groovy

```diff
@@ -158,6 +152,16 @@ class VividASTVisitor extends ClassCodeVisitorSupport {
         visitStatements = previousIsTestMethod
     }
 
+    @Override // This is overridden to avoid visiting annotations.
+    void visitAnnotations(AnnotatedNode node) {
+        for (AnnotationNode annotation : node.getAnnotations()) {
+            // skip built-in properties
+            if (!annotation.isBuiltIn()) {
+                //visitAnnotation(annotation);
```

Comment:
Why did you keep this commented-out code? If nothing needs to be done, should this method just be entirely blank?

Reply:
There really is no reason. The method should be empty. I just pushed the changes. 
Instead of the dead code there is now a message which also references this issue.